### PR TITLE
Add a selenium based scraping script.

### PR DESCRIPTION
**Pull request by awong-dev**

### CODE REVIEW

File: scrape.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        console.log(`Failed on (${x},${y}): ${err}`);
+        break;
+      }
+    }
+  }
+
+  console.log(JSON.stringify(data, null, 2));
+}
+
+module.exports = { setupPage, scrape };
```
Comment (jfuchs) [inline scrape.js]: OOC, why do you need setupPage exported?

**Review by jfuchs — APPROVED** [review]
I’ve never used selenium, so this is a pretty superficial review. Logically makes sense though.

File: scrape.js (hunk 1 above)
Comment (awong-dev) [reply scrape.js]: ah, cause I was loading it directly in the node.js interactive console. Selenium is ridiculously slow to start up so developing sanely requires being able to iterate without waiting for it all to spin up.

**Comment by awong-dev** [comment]
I updated it a bunch more. It's ugly code so going to just check in cause hacks gonna hack.

**Pull request merged**